lib/plugins/aws/provider.js

```diff
@@ -965,9 +1003,13 @@ class AwsProvider {
             },
             handler: { type: 'string' },
             image: {
-              type: 'string',
-              pattern:
-                '^\\d+\\.dkr\\.ecr\\.[a-z0-9-]+..amazonaws.com\\/([^@]+)|([^@:]+@sha256:[a-f0-9]{64})$',
+              anyOf: [
+                { $ref: '#/definitions/ecrImageUri' },
+                {
+                  type: 'string',
+                  pattern: '^[a-z]{1,32}$',
```

Comment:
It'll be nice to keep this pattern in variable, as e.g. we do here: https://github.com/serverless/serverless/blob/30015eafd2fb9d2e82d8f34ee8f10c1fb4e536a0/lib/configSchema.js#L3